mfhdf/hrepack/hrepack_list.c

```diff
@@ -386,6 +386,9 @@ list_vg(int32 infile_id, int32 outfile_id, int32 sd_id, int32 sd_out, int32 gr_i
                 printf("Error: Could not get name length for group with ref <%d>\n", ref);
                 goto out;
             }
+
+            if (vg_name != NULL)
+                free(vg_name);
```

Comment:
Freeing a NULL pointer is allowed by the standard. Checks for NULL are just noise.

Reply:
Right I knew that!!!